tests/microbenchmarks.py

```diff
@@ -4,19 +4,24 @@
 
 from utils import *
 
+
 def test_small_integers(test_case):
     print("Running", "test_small_integers")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
```

Comment:
At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils